scripts/dev_scripts/integration_tests.sh

```diff
@@ -11,8 +11,27 @@ COMPARE_DEPS=$WORKSPACE/tests/dependency_analyzer/compare_dependencies.py
 COMPARE_JSON_OUT=$WORKSPACE/tests/e2e/compare_e2e_result.py
 TEST_REPO_FINDER=$WORKSPACE/tests/e2e/repo_finder/repo_finder.py
 RUN_MACARON="python -m macaron -o $WORKSPACE/output"
+PYTHON="python "
 RESULT_CODE=0
 
+function _copy() {
+  # To replace all python commands when updating results, rather than comparing them.
+  # Note that this function will always attempt to perform a file copy operation on arg #2 and #3, if there are
+  # sufficient arguments.
+  if [ $# -eq 3 ] ; then
+    # Copy arg 2 to arg 3.
+    cp "$2" "$3"
+  fi
+}
+
+# Optional arguments for updating the expected results.
+if [ $# -ge 3 ] ; then
+  if [ $3 == "--update" ] ; then
+    echo "Updating expected results after running the tests needed to create them."
```

Comment:
```suggestion
    echo "Updating the expected results based on the previous run of integration tests."
```

Reply:
Macaron is still run to produce the output before the expected results are overwritten, so perhaps it should be something like:
"Updating the expected results to match those currently produced by Macaron."

Reply:
Yes, right. Sounds good :+1: